xchange-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingExchange.java

```diff
@@ -80,17 +81,11 @@ public boolean isAlive() {
     
     @Override
     public Observable<Object> connectionSuccess() {
-        if (privateStreamingService != null)
```

Comment:
Why was this removed?

Reply:
because I cannot resolve this, so going for the simplest option of just monitoring the public channel.